vmm/modules/vtd/vtd.c

```diff
@@ -349,6 +359,26 @@ static void vtd_invalidate_cache(UNUSED guest_cpu_handle_t gcpu, void *pv)
 	}
 }
 
+static void vmcall_activate_vtd(guest_cpu_handle_t gcpu UNUSED)
```

Comment:
should it be wrapped with ACTIVATE_VTD_BY_VMCALL? will it lead to "unused function" error when it is not compiled?